config/storybook.d.ts

```diff
@@ -0,0 +1,10 @@
+declare var module: any; // dangerous
+
+declare module "@kadira/storybook" {
+  interface Story {
+      add (storyName: string, callback: Function): Story;
+  }
+
+  export function storiesOf(name: string, module: any): Story;
+  export function action(name: string, ...params: any[]): Function;
```

Comment:
There's another API called [linkTo](https://github.com/kadirahq/react-storybook/blob/master/src/client/client_api.js#L67). Add that too.